tarantool/connection.py

```diff
@@ -533,8 +545,10 @@ def __init__(self, host, port,
         self.socket_timeout = socket_timeout
         self.reconnect_delay = reconnect_delay
         self.reconnect_max_attempts = reconnect_max_attempts
-        self.schema = Schema(self)
-        self.schema_version = 1
+        self.fetch_schema = fetch_schema
+        if self.fetch_schema:
+            self.schema = Schema(self)
```

Comment:
Let's set them to `None` otherwise

Reply:
Related with [this](https://github.com/tarantool/tarantool-python/pull/271#discussion_r1056184340).
If I understand correctly, to be able to make the connection again schema-full without reconnect, `None` is not needed here